l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -139,6 +140,9 @@
                     <field name="virtual_stamp" readonly="1"></field>
                     <field name="stamp_amount" readonly="1"></field>
                 </field>
+                <field name="invoice_line" position="attributes">
+                    <attribute name="context">{'form_view_ref' : 'l10n_it_fatturapa_in.view_invoice_line_form_fatturapa_in'}</attribute>
```

Comment:
Se si apre lo stesso form non va bene.
il fatto e' che il form di fatturapa aout https://github.com/eLBati/l10n-italy/pull/63/files#diff-02c1bad0d04b4b99db8693c9ce527616R11 
esplicita il campo readonly = 0
apposta.
forse ci sarebbe da sistemare anche i menu e le action che chiamano i form di fatturaPA clienti e fatturaPA fornitori, in modo che siano gestite perfettamente le viste da usare. sopratutto nel caso vengano installati entrambi.